src/app/view/components/form-email/form-email.component.html

```diff
@@ -3,9 +3,7 @@
   <input type="email" matInput formControlName="email" placeholder="Ex. mariorossi@gmail.com" />
   <mat-icon matSuffix>email</mat-icon>
   <mat-error *ngIf="emailForm.controls['email'].hasError('email') && !emailForm.controls['email'].hasError('required')">
-    Inserisci un indirizzo email valido
-  </mat-error>
-  <mat-error *ngIf="emailForm.controls['email'].hasError('required')">
-    L'email è <strong>obbligatoria</strong>
+    Please enter a valid email address
   </mat-error>
+  <mat-error *ngIf="emailForm.controls['email'].hasError('required')"> Email is <strong>required</strong> </mat-error>
```

Comment:
meglio scrivere "this field is required" una volta e riutilizzare la stringa che avere messaggi sempre diversi